raft/src/main/java/org/apache/kafka/raft/internals/VoterSetOffset.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.raft.internals;
+
+public class VoterSetOffset {
```

Comment:
Why did you add this type instead of using `LogHistory.Entry<VoterSet>`?

Reply:
I was on the fence about it, I thought this might be a bit more explicit/clear as a return type

Reply:
`LogHistory.Entry<VoterSet>` is explicit, no? It is a type that has an `offset()` and the `value()` of `VoterSet`.